src/pages/public_registration/action_accordions/DuplicateWarningModal.tsx

```diff
@@ -0,0 +1,92 @@
+import OpenInNewOutlinedIcon from '@mui/icons-material/OpenInNewOutlined';
+import { Box, Button, Typography } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+import { Link } from 'react-router-dom';
+import { Modal } from '../../../components/Modal';
+import { dataTestId } from '../../../utils/dataTestIds';
+import { getRegistrationLandingPagePath } from '../../../utils/urlPaths';
+
+interface DuplicateWarningModalProps {
+  isOpen: boolean;
+  toggleModal: () => void;
+  onConfirmNotDuplicate: () => void;
+  duplicateId: string | undefined;
+}
+
+export const DuplicateWarningModal = ({
+  isOpen,
+  toggleModal,
+  duplicateId,
+  onConfirmNotDuplicate,
+}: DuplicateWarningModalProps) => {
+  const { t } = useTranslation();
+
+  return (
+    <Modal
+      open={isOpen}
+      onClose={() => {
+        toggleModal();
+      }}
+      maxWidth="xs"
+      fullWidth={true}
+      headingText={t('registration.public_page.duplicate_warning_modal.headline')}
+      dataTestId={dataTestId.registrationLandingPage.duplicateRegistrationModal.duplicationModal}>
+      <Box sx={{ display: 'flex', flexDirection: 'column', gap: '1rem' }}>
+        <Typography>{t('registration.public_page.duplicate_warning_modal.publication_already_exists')}</Typography>
+        {duplicateId && (
+          <Box>
+            <Typography sx={{ display: 'inline' }}>
+              {`${t('registration.public_page.duplicate_warning_modal.dont_create_duplicates')} ${t('registration.public_page.duplicate_warning_modal.check_if')}`}
+            </Typography>
+            <Link
+              target="_blank"
+              data-testid={dataTestId.registrationLandingPage.duplicateRegistrationModal.duplicateRegistrationLink}
+              to={getRegistrationLandingPagePath(duplicateId)}>
+              <Box sx={{ gap: '0.25rem', display: 'inline-flex', mr: '0.25rem' }}>
+                <Typography
+                  sx={{ display: 'inline', textDecoration: 'underline', cursor: 'pointer', color: 'primary.light' }}>
+                  {t('registration.public_page.duplicate_warning_modal.this_publication')}
+                </Typography>
+                <OpenInNewOutlinedIcon
+                  sx={{
+                    cursor: 'pointer',
+                    color: 'primary.main',
+                    height: '1.3rem',
+                    width: '1.3rem',
+                    display: 'inline',
+                  }}
+                />
+              </Box>
+            </Link>
```

Comment:
Det er en del custom styling her på det som for meg ser ut som en standard link som skal se lik ut som alle andre linker.
Er dette en forenklet versjon av det samme?

```suggestion
            <MuiLink
              component={Link}
              target="_blank"
              data-testid={dataTestId.registrationLandingPage.duplicateRegistrationModal.duplicateRegistrationLink}
              to={getRegistrationLandingPagePath(duplicateId)}
              sx={{
                display: 'inline-flex',
                alignItems: 'center',
                gap: '0.25rem',
                mr: '0.25rem',
                color: 'primary.light',
              }}>
              {t('registration.public_page.duplicate_warning_modal.this_publication')}
              <OpenInNewOutlinedIcon fontSize="small" />
            </MuiLink>
```

Åpent spørsmål om linken skal følge farge til theme eller ha egen farge da, og om det skal være ikon. Men vi får bruke den informasjonen vi har

Reply:
Ja hvis det er sånn dere foretrekker å style linker ser jeg ikke noe problem med det.